megamek/src/megamek/client/bot/princess/NewtonianAerospacePathRanker.java

```diff
@@ -39,24 +39,26 @@ public NewtonianAerospacePathRanker(Princess owningPrincess) {
      */
     @Override
     public Entity findClosestEnemy(Entity me, Coords position, Game game) {
-        int range = 9999;
+        int range = Integer.MAX_VALUE;
```

Comment:
I'm just being nitpicky here, there should not be any difference between 9999 and MAX_INT value, but anyway.

Reply:
It's worth it regardless as I've ran tests with more than 10k units a side.